app/javascript/controllers/method_controller.js

```diff
@@ -0,0 +1,15 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = ["srcBody", "showSrcBodyButton"]
+
+  toggleShowSource(event) {
+    event.preventDefault()
+    this.showSrcBodyButtonTarget.classList.toggle("bg-blue-600")
+    this.showSrcBodyButtonTarget.classList.toggle("text-white")
+    this.showSrcBodyButtonTarget.classList.toggle("hover:bg-blue-700")
+    this.showSrcBodyButtonTarget.classList.toggle("hover:text-white-300")
+    this.showSrcBodyButtonTarget.classList.toggle("dark:bg-gray-900")
+    this.srcBodyTarget.classList.toggle("hidden")
+  }
+}
```

Comment:
I think we need for a newline at the end of files. I can create a PR with [EditorConfig](https://editorconfig.org/) file, if you wish, for convience.

Reply:
@AlexWayfer I am happy to add an editor config to file to help maintain consistency 👍 Do you want to make a PR to add that?

Reply:
> @AlexWayfer I am happy to add an editor config to file to help maintain consistency  Do you want to make a PR to add that?

Yeah, I can. Honestly, it's already unstaged in my local copy of the project, so it should not be difficult.